forge-gui/res/cardsfolder/upcoming/zenith_chronicler.txt

```diff
@@ -0,0 +1,7 @@
+Name:Zenith Chronicler
+ManaCost:2
+Types:Artifact Creature Phyrexian Construct
+PT:3/1
+T:Mode$ SpellCast | ValidCard$ Card.MultiColor | ValidActivatingPlayer$ Player | TriggerZones$ Battlefield | ActivatorThisTurnCast$ EQ1 | NoResolvingCheck$ True | Execute$ TrigDraw | TriggerDescription$ Whenever a player casts their first multicolored spell each turn, each other player draws a card.
+SVar:TrigDraw:DB$ Draw | Defined$ TriggeredCardOpponent
```

Comment:
I don't think this Defined works because all other players are not necessarily opponents of the triggered caster

Reply:
Huh my github desktop is acting strange and not applying my changes